content/en/docs/community/release_checklist.md

```diff
@@ -126,31 +158,34 @@ git tag --sign --annotate "$RELEASE_NAME" --message "Helm release $RELEASE_NAME"
 git push upstream "$RELEASE_NAME"
 ```
 
-This new tag is going to be the base for the patch release.
+This new tag is going to be the base for the patch release. Creating a
+[helm/helm milestone](https://github.com/helm/helm/milestones) is optional for
+patch releases.
 
 Make sure to check [helm on CircleCI](https://circleci.com/gh/helm/helm) to see
-that the release passed CI before proceeding.
+that the release passed CI before proceeding. Patch releases can skip steps 2-6
+and proceed to step 7 for [PGP signature](#7-pgp-sign-the-downloads).
 
-## 2. Change the Version Number in Git
+## 2. Major/Minor releases: Change the Version Number in Git
```

Comment:
I am not positive that step 2 is not included in the patch release process, but it does not appear that it should be. Input welcome.

Reply:
not necessary for patch releases.

Reply:
Ah, good to have confirmation.